pkg/storage/pod.go

```diff
@@ -117,6 +119,16 @@ func (s *podStorage) Store(newPods []PodMetricsPoint) {
 								"timestamp", newContainer.Timestamp)
 						}
 					}
+				} else if lastContainer, found := lastPod[newContainer.Name]; found && newContainer.StartTime.After(lastContainer.Timestamp) {
+					prevContainers[newContainer.Name] = ContainerMetricsPoint{Name: newContainer.Name, MetricsPoint: MetricsPoint{StartTime: newContainer.StartTime, Timestamp: newContainer.StartTime}}
```

Comment:
This doesn't pass all fields (e.g. memory). Please copy new metric point and override, timestamp and cpu usage.